tidy3d/web/webapi.py

```diff
@@ -337,7 +347,7 @@ def delete(task_id: TaskId) -> TaskInfo:
         Object containing information about status, size, credits of task.
     """
 
-    method = f"fdtd/task/{str(task_id)}"
+    method = f"tidy3d/projects/any/tasks/{task_id}"
```

Comment:
let's do it correctly.  the webapi looks strange.
use delete f"tidy3d/tasks/{task_id}/"